setup.py

```diff
@@ -3,16 +3,11 @@
 import os
 from setuptools import setup, find_packages
 from distutils.core import Extension
-import sys
 import platform
 
-INSTALL_REQUIRES = list()
-
-if sys.version_info[0] < 3:
-    avro = 'avro'
-    INSTALL_REQUIRES.extend(['futures', 'enum34', 'requests'])
-else:
-    avro = 'avro-python3'
+INSTALL_REQUIRES = ["futures; python_version<'3.3'",
+                    "enum34; python_version<'3.5'",
+                    'requests']
```

Comment:
suggesting using double-quotes for requests here to be consistent with the above quoting.